src/coffee/directives/api/map.coffee

```diff
@@ -1,28 +1,25 @@
+###globals angular,_,google###
 angular.module('uiGmapgoogle-maps.directives.api')
-.factory 'uiGmapMap', [
-  '$timeout', '$q','uiGmapLogger', 'uiGmapGmapUtil', 'uiGmapBaseObject',
-  'uiGmapCtrlHandle', 'uiGmapIsReady', 'uiGmapuuid',
-  'uiGmapExtendGWin', 'uiGmapExtendMarkerClusterer',
-  'uiGmapGoogleMapsUtilV3','uiGmapGoogleMapApi','uiGmapEventsHelper',
-  'uiGmapGoogleMapObjectManager'
-  ($timeout,$q, $log, GmapUtil, BaseObject,
-    CtrlHandle, IsReady, uuid,
-    ExtendGWin, ExtendMarkerClusterer,
-    GoogleMapsUtilV3,GoogleMapApi, EventsHelper, GoogleMapObjectManager) ->
-      'use strict'
+.factory 'uiGmapMap',
```

Comment:
ngAnnotate takes care of this now. I will be changing the other directives, factories as I go.